docs/src/main/asciidoc/transaction-guide.adoc

```diff
@@ -113,6 +113,13 @@ public class SantaClausService {
 <1> Inject the `TransactionManager` to be able to activate `setRollbackOnly` semantic.
 <2> Programmatically decide to set the transaction for rollback.
 
+
+=== Transaction Configuration
```

Comment:
Asciidoc requires space after a section title. Some parsers are more lenient but let's be safe.

Reply:
done